### PR TITLE
Make TGW support User Services account

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##
This PR adds Transit Gateway support for the User Services account.

## 💭 Motivation and context ##
We want the User Services account to be able to route data through the Shared Services Transit Gateway.  In order to accomplish that, we create the resources specified in this PR.

## 🧪 Testing ##
These changes were successfully applied in Staging.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

### CODE REVIEW

File: transitgateway.tf [hunk 1]
```diff
@@ -66,15 +66,15 @@ resource "aws_ram_principal_association" "tgw" {
 resource "aws_ec2_transit_gateway_route_table" "tgw_attachments" {
   provider = aws.sharedservicesprovisionaccount
 
-  for_each = merge(local.env_accounts_same_type, local.pca_account_same_type)
+  for_each = merge(local.env_accounts_same_type, local.pca_account_same_type, local.userservices_account_same_type)
```
Comment (jsf9k) [inline transitgateway.tf]: No! We want User Services to use the default TGW route table just like Shared Services does. As as add assessment environments, VPN tunnels, etc. new routes for these things will automatically propagate to the default route table. This is precisely what we want for the Shared Services and User Services.

**Review by jsf9k — CHANGES_REQUESTED** [review]
User Services should use the default TGW route table.

File: transitgateway.tf (hunk 1 above)
Comment (dav3r) [reply transitgateway.tf]: Roger- see https://github.com/cisagov/cool-sharedservices-networking/pull/37/commits/3bc6a4d916fdce8cee81288453cc3ed54f071ea3.

**Review by jsf9k — APPROVED** [review]

**Review by hillaryj — APPROVED** [review]
🎉

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔

**Pull request merged**